### ISSUE TITLE
Importing kubespawner fails with kubernetes python package version 21.7.0

### ISSUE THREAD

**Issue by hobackskiier** [jupyterhub/kubespawner#556]
### Bug description
version `21.7.0` of the `kubernetes` package was released yesterday - https://pypi.org/project/kubernetes/#history

When installing the `jupyterhub-kubespawner` package, it now defaults to installing this version as a dependency.

Something in this version seems to have broken `kubeswpaner`, when importing the package, the following error is seen:

```
>>> import kubespawner
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/kubespawner/__init__.py", line 13, in <module>
    from kubespawner.spawner import KubeSpawner
  File "/usr/local/lib/python3.8/dist-packages/kubespawner/spawner.py", line 43, in <module>
    from .objects import make_namespace
  File "/usr/local/lib/python3.8/dist-packages/kubespawner/objects.py", line 16, in <module>
    from kubernetes.client.models import V1EndpointPort
ImportError: cannot import name 'V1EndpointPort' from 'kubernetes.client.models' (/usr/local/lib/python3.8/dist-packages/kubernetes/client/models/__init__.py)
```

#### Expected behaviour
`kubespawner` imports without error

#### Actual behaviour
import fails due to an error importing `V1EndpointPort'` from `kubernetes.client.models`

### How to reproduce
```
pip install jupyterhub-kubespawner
```

open a python shell

```
import kubespawner
```

**Comment by ktaletsk** [comment]
Temporary fix: install the last compatible version of kubernetes alongside jupyterhub-kubespawner: `pip install kubernetes==20.13.0`.

More long-term protection from upstream changes would be to add the upper limit on kubernetes package verison here: https://github.com/jupyterhub/kubespawner/blob/d913841afb59f1b9a4e1f8c69ada2189ee34729f/setup.py#L23

**Comment by athornton** [comment]
I think this will fix the problem: https://github.com/jupyterhub/kubespawner/pull/558

**Comment by consideRatio** [comment]
I'd love a verification that #558, now merged, was sufficient to make things work!

**Comment by memark** [comment]
@consideRatio It works. I verified it by running:

```
docker run --rm -it python
pip install git+https://github.com/jupyterhub/kubespawner.git
python
import kubespawner
```

and it works. (If running just `pip install jupyterhub-kubespawner`, like OP did, it fails.)

**Comment by consideRatio** [comment]
Closed by #558
